From 236a0c578369bfc726f7d9a830975cbeaefbfee5 Mon Sep 17 00:00:00 2001
From: Nick Schermer <nick@xfce.org>
Date: Mon, 23 Jul 2012 15:31:48 +0000
Subject: Remove FORTIFY_SOURCE=2.

This gives warnings with full debug builds on recent gcc versions.
---
diff --git a/m4macros/xdt-features.m4 b/m4macros/xdt-features.m4
index c5f5583..f258822 100644
--- a/m4macros/xdt-features.m4
+++ b/m4macros/xdt-features.m4
@@ -79,7 +79,7 @@ AC_HELP_STRING([--disable-debug], [Include no debugging support]),
                               -Wcast-align -Wformat-security \
                               -Winit-self -Wmissing-include-dirs -Wundef \
                               -Wmissing-format-attribute -Wnested-externs"
-    CPPFLAGS="$CPPFLAGS -D_FORTIFY_SOURCE=2"
+    CPPFLAGS="$CPPFLAGS"
 
     if test x`uname` = x"Linux"; then
       xdt_cv_additional_CFLAGS="$xdt_cv_additional_CFLAGS -fstack-protector"
--
cgit v0.9.2
